templates/AureliaSpa/README.md

```diff
@@ -0,0 +1,40 @@
+# Welcome to ASP.NET 5
```

Comment:
This file is out of date. Could you update it? e.g., take content from https://github.com/aspnet/JavaScriptServices/blob/dev/templates/Angular2Spa/README.md